ui/src/components/VolumeListTable.js

```diff
@@ -536,17 +186,52 @@ const VolumeListTable = (props) => {
   };
 
   return (
-    <VolumeListContainer>
+    <Box height={'100%'}>
       <Table
         columns={columns}
         data={volumeListData}
-        nodeName={nodeName}
-        onClickRow={onClickRow}
-        volumeName={volumeName}
-        theme={theme}
-        isSearchBar={isSearchBar}
-      />
-    </VolumeListContainer>
+        defaultSortingKey={'health'}
+        getRowId={(row) => row.name}
+      >
+        <Box
+          display="flex"
+          justifyContent={'space-between'}
+          pt={'16px'}
+          px={'16px'}
+        >
+          <Table.SearchWithQueryParams
+            displayedName={{
+              singular: 'volume',
+              plural: 'volumes',
+            }}
+          />
+          <Button
+            variant={'primary'}
+            label={intl.formatMessage({ id: 'create_new_volume' })}
+            icon={<i className="fas fa-plus" />}
+            onClick={() => {
+              // depends on if we add node filter
+              if (nodeName) {
```

Comment:
To be checked if the property is still used but it seems not so maybe this can be deleted